masterlist.yaml

```diff
@@ -24276,19 +24579,19 @@ plugins:
   - name: 'NPO Module - Crossbows.esp'
     after:
       - 'PerkusMaximus_Warrior.esp'
-
   - name: 'Default Teammate Weapon Draw Distance.esp'
     after:
       - 'Convenient Horses.esp'
-
   - name: 'Undeath.esp'
-    url: [ 'http://www.nexusmods.com/skyrim/mods/40607' ]
```

Comment:
Same reason as in [#136 (comment)](https://github.com/loot/skyrim/pull/136#discussion_r122197248)